src/main/java/Duke.java

```diff
@@ -1,10 +1,190 @@
+import java.util.Scanner;
+//import java.util.Arrays;
+
 public class Duke {
+
+    private static final Scanner input = new Scanner(System.in);
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput = "";
+        Task[] currentTasks = new Task[100];
+        int numOfTasks = 0;
+
+        showWelcomeMessage();
+
+        while(!userInput.equals("bye")) {
+            userInput = getUserInput();
+            String[] splitUserInput = splitCommands(userInput);
+
+            switch (splitUserInput[0]) {
+
+                case "bye":
+                    exitProgram();
+                    break;
+
+                case "list":
+                    commandList(currentTasks, numOfTasks);
+                    break;
+
+                case "done":
+                    markAsDone(currentTasks, numOfTasks, splitUserInput[1]);
+                    break;
+
+                case "deadline":
+                    numOfTasks = commandDeadline(currentTasks, numOfTasks, splitUserInput[1]);
+                    break;
+
+                case "todo":
+                    numOfTasks = commandToDo(currentTasks, numOfTasks, splitUserInput);
+                    break;
+
+                case "event":
+                    numOfTasks = commandEvent(currentTasks, numOfTasks, splitUserInput[1]);
+                    break;
+
+                case "help":
+                    commandHelp();
+                    break;
+
+                default:
+                    invalidCommand();
```

Comment:
There are several unnecessary empty lines here which could be removed.

Reply:
Thank you for your inputs Xuche!